test/UnitTest/Microsoft.AspNetCore.OData.Test/Extensions/Endpoint/ODataEndpointRouteValueTransformerTests.cs

```diff
@@ -104,6 +107,73 @@ public void TransformAsyncReturnsCorrectRouteValues()
             Assert.Same(actionDescriptor, httpContext.ODataFeature().ActionDescriptor);
         }
 
+        [Theory]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
```

Comment:
>false [](start = 20, length = 5)

don't really have to pass this in, since it's false in all cases.

Reply:
resolved